src/Lumina/Data/Files/TexFile.cs

```diff
@@ -166,18 +166,14 @@ public unsafe struct TexHeader
             [FieldOffset( 12 )]
             public ushort Depth;
 
-            /// <summary>
-            /// The field has been repurposed; use <see cref="MipLevelsCount"/>.
-            /// </summary>
+            /// <summary>The field has been repurposed; use <see cref="MipCount"/>.</summary>
             [FieldOffset( 14 )]
-            [Obsolete( $"Use {nameof( MipLevelsCount )} instead; the field has been repurposed as two fields." )]
+            [Obsolete( $"Use {nameof( MipCount )} instead; the field has been repurposed as three fields." )]
```

Comment:
this has been obsolete for a while, does this just become a breaking change?

Reply:
Nothing in the PR should break any existing code (from getting JITed), but I guess it's alright to get rid of that obsolete field for sure. Remove `ushort MipLevels` field?

Reply:
yeah that's what i was thinking, its been around for ages and not really serving any purpose anymore